setup.py

```diff
@@ -77,12 +77,17 @@ def get_extra_deps(rel_path):
     classifiers=CLASSIFIERS,
     url="https://microsoft.github.io/Olive/",
     download_url="https://github.com/microsoft/Olive/tags",
-    packages=find_packages(exclude=("test", "examples*")),
+    packages=find_packages(include=["olive*"]),
     python_requires=">=3.8.0",
     install_requires=requirements,
     extras_require=EXTRAS,
-    include_package_data=True,
-    package_data={},
+    include_package_data=False,
```

Comment:
Default value is `False`. Maybe we can remove it.

Reply:
I make it explicitly for clarity.